src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -1003,6 +1003,11 @@ protected override bool TypeSymbolEquals(TypeSymbolWithAnnotations other, TypeCo
 
                 return base.TypeSymbolEquals(other, comparison);
             }
+
+            public override string ToDisplayString(SymbolDisplayFormat format)
+            {
+                return _underlying.TypeSymbol.ToDisplayString(format) + "?";
```

Comment:
>"?" [](start = 72, length = 3)

I think this could be a nullable value type, it would not be appropriate to add "?" in that case.